### PR TITLE
Assignment1

### PR DESCRIPTION
**Pull request by Ashok-Ab4**
Run Assignment1_bash.sh to run the python code!

### CODE REVIEW

**Comment by kratipatidar** [comment]
The code seems accurate. Just wanted to clarify if the train-test split is required by the assignment or that's been done for the random forest classifier.

**Comment by Ashok-Ab4** [comment]
> The code seems accurate. Just wanted to clarify if the train-test split is required by the assignment or that's been done for the random forest classifier.

Not a part of the assignment. Just added it in to make it cleaner. Let me know if any changes need to be made!

**Review by kratipatidar — APPROVED** [review]
Okay! That's it then.

File: Assignment1.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.metrics import accuracy_score
+from sklearn.model_selection import train_test_split
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import Normalizer
+
+
+def main():
+    column_id = ["Sepal length", "Sepal width", "Petal length", "Petal width", "class"]
+    iris_df = pd.read_csv("Iris.data", names=column_id)
```
Comment (dafrenchyman) [inline Assignment1.py:17]: Data wasn't here to load.

**Review by dafrenchyman — APPROVED** [review]
You lost points because I had to add the data to the project (it didn’t run on first try). The output for the missing features was incorrect too. But I liked the plots (especially the 3D one). You also did a train/test split even though you didn’t need to. Great job! I gave some extra points for the bash script too.

**Pull request merged**